conda/recipes/libcudf/install_libcudf_example.sh

```diff
@@ -1,4 +1,14 @@
 #!/bin/bash
-# Copyright (c) 2018-2022, NVIDIA CORPORATION.
+# Copyright (c) 2018-2024, NVIDIA CORPORATION.
 
-./cpp/examples/build.sh
+# store old CMAKE_INSTALL_PREFIX if set
+#CMAKE_INSTALL_PREFIX_OLD=${CMAKE_INSTALL_PREFIX}
```

Comment:
Is this logic to store the old value necessary? I think this environment variable would only affect this script's environment, and not other scripts.

Reply:
Nah, not necessary, I was trying to locally experiment which got committed.

Reply:
Not necessary at all. Some local testing artifacts seeped through here.